pom.xml

```diff
@@ -24,7 +24,8 @@
   </scm>
   <properties>
     <quarkus.version>3.3.3</quarkus.version>
-    <poi.version>5.2.4</poi.version>
+    <poi.version>5.2.5</poi.version>
+    <compress.version>1.25.0</compress.version>
```

Comment:
I suppose this version needs to be in sync with what is used by POI, maybe add a comment here so we don't forget?